local/bin/py/build/update_pre_build.py

```diff
@@ -42,6 +42,17 @@ def __init__(self, tempdir):
         self.extract_dir = "{0}".format(
             join(self.tempdir, "extracted") + sep
         )
+
+        self.apw_integrations = [
+            {
+                'extract_path': f'{self.extract_dir}/dogweb/integration/akamai_mpulse',
+                'cache_path': 'temp/content/en/integrations/akamai_mpulse.md'
+            },
+            {
+                'extract_path': f'{self.extract_dir}/integrations-extras/ably',
+                'cache_path': 'temp/content/en/integrations/ably.md'
+            }
+        ]
```

Comment:
I'm wondering if we can just build a list of the names instead and do any path work in the usage logic instead.
For example the cache path will always be the same also the extract path may need to be removed from all repos not just a specific integrations-extras etc as we still have duplicates

```
self.apw_integrations = [
'akamai_mpulse',
'ably'
]
```

Reply:
thanks @davidejones!    refactored this